tlUtils/config.js

```diff
@@ -0,0 +1,57 @@
+export const MAINNET_RPC_URL = `https://mainnet.infura.io/v3/9344a875cace468ca6c8029147cda56a`
+export const GOERLI_RPC_URL = `https://goerli.infura.io/v3/9344a875cace468ca6c8029147cda56a`
```

Comment:
I think we'll need to create repository or vercel secrets for these Project IDs.